FinnUI/Sources/Fullscreen/Settings/Foundation/Views/BasicListCell.swift

```diff
@@ -7,24 +7,33 @@ import FinniversKit
 
 @available(iOS 13.0.0, *)
 public struct BasicListCell: View {
+    var action: (() -> Void)?
     private let model: BasicTableViewCellViewModel
     private let title: Text
     private let subtitle: Text?
     private let detailText: Text?
 
     public init(
         model: BasicTableViewCellViewModel,
+        action: (() -> Void)? = nil,
         @ViewBuilder title: (String) -> Text = title,
         @ViewBuilder subtitle: (String?) -> Text? = subtitle,
         @ViewBuilder detailText: (String?) -> Text? = detailText
     ) {
         self.model = model
+        self.action = action
         self.title = title(model.title)
         self.subtitle = subtitle(model.subtitle)
         self.detailText = detailText(model.detailText)
     }
 
     public var body: some View {
+        Button(action: { self.action?() }) {
+            content
+        }.buttonStyle(BasicButtonStyle())
```

Comment:
It's possible to write this like

```swift
Button(content, action: { self.action?() })
  .buttonStyle(BasicButtonStyle())
```

just as a style option

Reply:
I think it's a different initialiser that requires `StringProtocol`, while `content` here is `some View`.

Reply:
right :P